R/slackr_utils.R

```diff
@@ -89,6 +89,23 @@ slackr_users <- function(bot_user_oauth_token=Sys.getenv("SLACK_BOT_USER_OAUTH_T
 
 }
 
+stop_for_status <- function(r) {
```

Comment:
Can we add a header to this function like the other functions have and get rid of the explicit function calls? i.e.:

```
#' stop_for_status
#' 
#' some_description
#' @importFrom httr stop_for_status content status_code
#' @param r the response
#' @return what_will_this_return? do we want it to return the content of the response or a warning? or do we want to return null invisibly? 
```

Reply:
This should be an internal function (not exported).  In my experience it's best not to have the roxygen template for internal functions, because it tends to get included in the package help (which it should not be).

But let me make your requested changes and we can take it from there.

Reply:
I've pushed the requested changes.

Reply:
you can avoid publishing the internal function docs to the package help with @noRd I believe. I'll give this a look in a bit. Thanks again!

Reply:
Top tip!  Yes, that should work.  I'll apply the fix.

Reply:
Done.